core/templates/external-js/save-offline-external-js.tid

```diff
@@ -4,5 +4,6 @@ title: $:/core/save/offline-external-js
 \define saveTiddlerFilter()
 [is[tiddler]] -[prefix[$:/state/popup/]] -[prefix[$:/temp/]] -[prefix[$:/HistoryList]] -[status[pending]plugin-type[import]] -[[$:/core]] -[[$:/plugins/tiddlywiki/filesystem]] -[[$:/plugins/tiddlywiki/tiddlyweb]] -[[$:/boot/boot.css]] -[type[application/javascript]library[yes]] -[[$:/boot/boot.js]] -[[$:/boot/bootprefix.js]] +[sort[title]] $(publishFilter)$
 \end
-\define coreURL() tiddlywikicore-$(version)$.js
+<$let coreURL={{$:/core/save/offline-external-js/coreURL}}>
```

Comment:
Sorry. .. I didn't see https://github.com/simonbaird/tiddlyhost/issues/171#issuecomment-1340262616 as a wrote my comment above. ... So it may be OK to ignore it